tests/keras_tests/feature_networks_tests/feature_networks/linear_collapsing_test.py

```diff
@@ -125,5 +125,107 @@ def create_networks(self):
+        x = layers.DepthwiseConv2D(kernel_size=(5, 5), strides=(1, 1), padding='same',
+                                   use_bias=False)(x)
+        x = tf.math.add(x, tf.constant(np.random.normal(size=x.shape[-1]), dtype=x.dtype))
+
+        # #################
+        # Conv2DTranspose #
+        # #################
+        # Collapse Conv2DTranspose with bias
+        x = layers.Conv2DTranspose(filters=9, kernel_size=(5, 5), strides=(1, 1), padding='same',
+                                   use_bias=True, bias_initializer='glorot_uniform')(x)
+        x = tf.math.add(x, tf.constant(np.random.normal(size=x.shape[-1]), dtype=x.dtype))
+        x = layers.ReLU()(x)
+
+        # Collapse Conv2DTranspose without bias, const first argument of tf.math.add
+        x = layers.Conv2DTranspose(filters=9, kernel_size=(5, 5), strides=(1, 1), padding='same',
+                                   use_bias=False)(x)
+        x = tf.math.add(tf.constant(np.random.normal(size=x.shape[-1]), dtype=x.dtype), x)
+        x = layers.ReLU()(x)
+
+        # Collapse + operator to Conv2DTranspose without bias
+        # TODO: replace add with + (currently using tf.math.add because below TF 2.14 creates TFOpLambda which fails ths node matcher)
+        x = layers.Conv2DTranspose(filters=9, kernel_size=(5, 5), strides=(1, 1), padding='same',
+                                   use_bias=False)(x)
+        x = tf.math.add(x, tf.constant(np.random.normal(size=x.shape[-1]), dtype=x.dtype))
+
+        # #######
+        # Dense #
+        # #######
+        x = layers.Reshape((-1,))(x)
+        # Collapse Dense with bias
+        x = layers.Dense(9, use_bias=True, bias_initializer='glorot_uniform')(x)
+        x = tf.math.add(x, tf.constant(np.random.normal(size=x.shape[-1]), dtype=x.dtype))
+        x = layers.ReLU()(x)
+
+        # Collapse Dense without bias, const first argument of tf.math.add
+        x = layers.Dense(9, use_bias=False)(x)
+        x = tf.math.add(tf.constant(np.random.normal(size=x.shape[-1]), dtype=x.dtype), x)
+        x = layers.ReLU()(x)
+
+        # Collapse + operator to Conv2DTranspose without bias
+        # TODO: replace add with + (currently using tf.math.add because below TF 2.14 creates TFOpLambda which fails ths node matcher)
+        x = layers.Dense(9, use_bias=False)(x)
+        x = tf.math.add(x, tf.constant(np.random.normal(size=x.shape[-1]), dtype=x.dtype))
+
+        # Don't collapse
+        x2 = layers.Dense(9, use_bias=True, bias_initializer='glorot_uniform')(x)
+        x = tf.math.add(x2, x)
+        y = layers.ReLU()(x)
+
+        return tf.keras.models.Model(inputs=inputs, outputs=y)
+
+    def compare(self, quantized_model, float_model, input_x=None, quantization_info=None):
+        super().compare(quantized_model, float_model, input_x, quantization_info)
+        for layer in quantized_model.layers:
+            if type(layer) in [layers.Conv2D, layers.DepthwiseConv2D, layers.Conv2DTranspose, layers.Dense]:
```

Comment:
you can save the const in the init function and then check here if it equals to the bias (when bias is none).